### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by kalbermattenm**
Maybe more to come

### CODE REVIEW

**Comment by monodo** [comment]
You might want to use black: https://pypi.org/project/black/ for code formatting

**Comment by kalbermattenm** [comment]
Hmmmm... I just brought it back to some kind of minimalist application. We should definitely implement some logging, but we should rather use the default logging engines, either from Flask, either from the default logging module from Python.

@monodo, could you please review that?

File: pytree/views.py [hunk 1]
```diff
@@ -117,4 +105,3 @@ def add(s, attribute):
     @staticmethod
     def fromName(name):
         return getattr(PointAttributes, name)
-#%%
```
Comment (nicolas-sitylb) [inline pytree/views.py:120]: Git will probably complain if there is no new line at the EOF.

File: pytree/views.py [hunk 2]
```diff
@@ -82,18 +73,15 @@ def profile_config_gmf2():
...
     return jsonify(vars)
-#%%
+
 
 class PointAttribute:
     def __init__(s, name, elements, bytes):
         s.name = name
         s.elements = elements
         s.bytes = bytes
-#%%
```
Comment (nicolas-sitylb) [inline pytree/views.py:96]: These are Jupyter cells, they are pretty nice to have when developing, but you can safely remove them.

File: .env.sample [hunk 3]
```diff
@@ -1,3 +1,3 @@
 DEPLOY_ENV=DEV
-PORT=8888
+PORT=6001
```
Comment (nicolas-sitylb) [inline .env.sample:2]: This is just a sample file, you probable don't want to expose your actual PORT to the world.

File: pytree/views.py [hunk 4]
```diff
@@ -2,16 +2,17 @@
...
 import yaml
 from yaml import FullLoader
 from flask_cors import cross_origin
 
-yaml_config_file = (dirname(abspath(__file__)) + ".yml")
+yaml_config_file = (path.dirname(path.abspath(__file__)) + ".yml")
 with open(yaml_config_file, 'r') as f:
     pytree_config = yaml.load(f, Loader=FullLoader)
 
+import logging
```
Comment (nicolas-sitylb) [inline pytree/views.py]: 👍🏼

**Review by nicolas-sitylb — CHANGES_REQUESTED** [review]

File: .env.sample (hunk 3 above)
Comment (kalbermattenm) [reply .env.sample:2]: No, I am not exposing anything, I am just making it compilant with what is currently written in the `USAGE` section of the README file...

File: pytree/views.py (hunk 4 above)
Comment (kalbermattenm) [reply pytree/views.py]: Actually, this is a left-over. Logging is deactivated in the current state and I won't have time to do something about it right now...

**Comment by kalbermattenm** [comment]
Thanks!

**Pull request merged**